about/package-scope.md

```diff
@@ -30,9 +28,229 @@ open a [pre-submission inquiry using a GitHub Issue](https://github.com/pyOpenSc
 one of our editors. We are happy to look at your package and help you understand
 whether it is in scope or not.
 
-```{include} /appendices/scope.md
+## About the scope of packages that we review
+
+pyOpenSci supports packages that support both open reproducible science,
+data processing and and the various stages of managing the
+data lifecycle. Packages submitted to pyOpenSci should fit into one or
+more of the categories below.
```

Comment:
```suggestion
more of the categories below and should be within our technical scope.
```

Based on the header level and the language I understand this section be a segue into talking about two things: "categories" and "technical scope". If that's correct than maybe rephrase to be a bit higher level, "I'm about to tell you about two things that both are part of our scope". As it's currently written I'm not quite clear why we need both this section and the previous one since they both talk about scope generally and include a callout to the presubmisson form. (I think you want to make sure that's the first thing someone reads before you give a detailed description of scope?)

Reply:
ok so i'm reworking this. then maybe you can do another top to bottom review and see if it makes more sense / edit as needed?!